### ISSUE TITLE
[BottomNavigation] Clean up static constant names

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#6720]
Many of the static constant values have long prefixes. Since they're statically scoped to the implementation file, they don't require prefixes.  For example, `kMDCBottomNavigationBarIsAccessibilityElement` should be `kIsAccessibilityElement`.

---
#### Internal data
- Associated internal bug: [b/126912860](http://b/126912860)
